pkg/lightning/backend/external/concurrent_reader.go

```diff
@@ -87,13 +86,7 @@ func (r *concurrentFileReader) read(bufs [][]byte) ([][]byte, error) {
 				buf,
 			)
 			if err != nil {
-				log.FromContext(r.ctx).Error(
```

Comment:
it prints too many logs when context cancel, it's not needed, there is no further usefully info in it.